bin/check_dl.sh

```diff
@@ -2,9 +2,9 @@
 set -e
 pip install tifffile #TODO: check if really required
 
-wget https://www.dropbox.com/s/0rvuae4mj6jn922/isbi.tar.gz -O ./data/isbi.tar.gz
+wget -P data -N https://www.dropbox.com/s/0rvuae4mj6jn922/isbi.tar.gz
 tar -xf ./data/isbi.tar.gz -C ./data/
-(set -e; for f in examples/_tests_scripts/{dl,z}*.py; do PYTHONPATH=./catalyst:${PYTHONPATH} python "$f"; done)
+(set -e; for f in examples/_tests_scripts/{dl,z_dl}*.py; do PYTHONPATH=./catalyst:${PYTHONPATH} python "$f"; done)
```

Comment:
Sorry, but we have to check all our integration scripts, both DL and RL.

Reply:
Whatever, but the naming is unclear then and dl tests require dependencies beyond requirements. 
There are separrate CI stages for rl, why rl and dl test are still mixed?

Reply:
cause it's much easier and faster to check them both in one place

Reply:
you can rename script to `check_dl_rl_scripts`, nevertheless we need all test checks

Reply:
Okay, but this should probably be done in another PR, not by me. I reverted my changes related to this issue.